.ci/ci.yml

```diff
@@ -62,6 +62,7 @@ jobs:
 - job: swig
   displayName: Linux SWIG Interface
   dependsOn: libshogun
+  timeoutInMinutes: 90
```

Comment:
as well? :D

Reply:
it should be unrelated. It's just with the python notebook job it can go just above 60 minutes, so just set it to 90